Source/Android/src/org/dolphinemu/dolphinemu/gamelist/GameListFragment.java

```diff
@@ -128,6 +128,7 @@ public void onListItemClick(ListView listView, View view, int position, long id)
 		Intent intent = new Intent(getActivity(), EmulationActivity.class);
 		intent.putExtra("SelectedGame", item.getPath());
 		startActivity(intent);
+		startForeground();
```

Comment:
This is incorrect entirely. startForeground isn't part of the ListFragment or the Fragment class it inherits from.